tools/fill_conf_values.py

```diff
@@ -36,12 +37,76 @@ def hash_filter(string, htype):
     return h.hexdigest()
 
 
+def nginx_brotli_installed():
+    """Check if the nginx brotli module is installed
+
+    Returns
+    -------
+    installed : bool
+        True if the nginx brotli module is installed, False otherwise
+    dynamic : bool
+        True if the module is dynamically loaded, False otherwise
+    modules_dir : str
+        The directory where the nginx modules are located if dynamic loading is used
+    """
+
+    installed = False
+    dynamic = False
+    dir = None
+
+    try:
+        output = subprocess.check_output(
+            ["nginx", "-V"], stderr=subprocess.STDOUT
+        ).decode("utf-8")
+        if (
+            "--add-module" in output
+            and "brotli" in output.split("--add-module")[1].strip()
+        ):
+            installed = True
+        elif (
+            "--add-dynamic-module" in output
+            and "brotli" in output.split("--add-dynamic-module")[1].strip()
+        ):
+            installed = True
+            dynamic = True
+            # we try to figure out where the modules are, there are 2 possibilities
+            # 1. the --modules-path directive is used
+            # 2. the default directory is used, which is where the configuration file is located so we look for it
+            if "--modules-path" in output:
```

Comment:
nginx has a search path, so will

```
load_module modules/ngx_http_brotli_filter_module.so;
load_module ...
```

not suffice?

Reply:
Well I actually tried that on my linux machine, and it didn't work! Not sure why... but it did look into the directory where the config is and didn't look at any other path.

Reply:
Do you have a minimal config you use to test with?

Reply:
What do you mean? I basically run SkyPortal with that modified version of baselayer. I ran it on an ubuntu machine with 8GB of ram, that had nginx but that I replaced with these instructions.